module/handler/info_handler.py

```diff
@@ -89,6 +91,16 @@ def handle_combat_low_emotion(self):
 
         return self.handle_popup_confirm('IGNORE_LOW_EMOTION')
 
+    def handle_use_data_key(self):
+        if not self.appear(POPUP_CONFIRM) and not self.appear(POPUP_CANCEL):
+            return False
+
+        ocr_result = DATA_KEY_USE.ocr(self.device.image)
+        if 'Data Key' in ocr_result:
+            return self.handle_popup_confirm('DATA_KEY_USE')
+
+        return False
+
```

Comment:
Add `USE_DATA_KEY = False`  in config class, set it to True when running war achieves.
In `handle_use_data_key()`, use this to skip the whole detection process when not running war achieves.
```
if not self.config.USE_DATA_KEY:
    return False
```

Reply:
I had switched OCR because the `self.appear` sometimes failed to evaluate to true but just noticed that uses `button.appear_on` rather than `button.match`, so I'll try using `offset=1` to see if that makes a difference.

Reply:
You can use `self.appear()` with `offset=(20, 20)`, `offset=True` equals `offset=(0, 30)`